src/components/FarmingList/index.js

```diff
@@ -223,46 +223,53 @@ function FarmingList({ campaigns, color, disbaleLinks, maxItems = 10 }) {
               justifyContent={"space-between"}
               marginBottom={"2px"}
               flexDirection={"row"}
+              style={{ margin: "auto" }}
             >
               <TokenLogo
                 address={pairData.stakablePair.token0.id}
                 size={"13px"}
                 defaultText={pairData.stakablePair.token0.symbol}
+                flexBasis={"30%"}
+                justifyContent="flex-end"
               />
-              <DataText>
+              <DataText flexBasis={"30%"} textAlign="right" justifyContent="flex-end">
                 {formattedNum(pairData.stakablePair.reserve0)}
               </DataText>
               <FormattedName
                 text={
                   nativeCurrencyWrapper.symbol ===
-                  pairData.stakablePair.token0.symbol
+                    pairData.stakablePair.token0.symbol
                     ? nativeCurrency
                     : pairData.stakablePair.token0.symbol
                 }
                 maxCharacters={below600 ? 8 : 16}
                 adjustSize={true}
                 link={true}
+                style={{ flexBasis: "30%", textAlign: "left" }}
               />
             </AutoRow>
-            <AutoRow justifyContent={"space-between"} flexDirection={"row"}>
+            <AutoRow justifyContent={"space-between"} flexDirection={"row"} style={{ margin: "auto" }}>
               <TokenLogo
                 address={pairData.stakablePair.token1.id}
                 size={"13px"}
                 defaultText={pairData.stakablePair.token1.symbol}
+                flexBasis={"30%"}
+                justifyContent="flex-end"
               />
-              <DataText>
+              <DataText flexBasis={"30%"} textAlign="right" justifyContent="flex-end">
                 {formattedNum(pairData.stakablePair.reserve1)}
               </DataText>
               <FormattedName
                 text={
                   nativeCurrencyWrapper.symbol ===
-                  pairData.stakablePair.token1.symbol
+                    pairData.stakablePair.token1.symbol
                     ? nativeCurrency
                     : pairData.stakablePair.token1.symbol
                 }
                 maxCharacters={below600 ? 8 : 16}
                 adjustSize={true}
                 link={true}
+                style={{ flexBasis: "30%", textAlign: "left" }}
```

Comment:
I don't think its good to use inline styles in this way flex-basis in particular it would be good if we can do it some other way since its not DRY and easy to mainatin...